### DISCUSSION TITLE
dot-commands

### DISCUSSION THREAD

**Discussion by irameez** [pawelsalawa/sqlitestudio#4184] [Q&A]
Hello,
I have looked at the documentation but couldn't find the answer to this, is it possible to use SQLite dot commands in SQLiteStudio.

Rameez.

**Comment by pawelsalawa** [comment]
In the GUI (the main) application - no. Dot commands are not SQL standard and are unique to each client (if it has same, others, or any at all).

SQLiteStudio has (experimental!) command line client, which is distributed alongside of the GUI application. Its name depends on the operating system, but its placed next to the GUI executable and has the "cli" suffix.

Once you run the CLI of SQLiteStudio, you have various dot commands there, but very little of them match the ones from official SQLite client. SQLiteStudio CLI has different ones. You can type `.help` to learn them and then `.help dblist` (or any other command) to learn more details about it.

**Reply by irameez** [reply]
Ok, thanks for the reply.
